### ISSUE TITLE
image segmentation with labelimg

### ISSUE THREAD

**Issue by essalahsouad** [tzutalin/labelImg#460]
Hi,
I want to know I can use labelimg for image segmentation .

**Comment by kashish2803** [comment]
Nope! It can only be used to annotate images for YOLO and pascal format. You probably need to use libraries like opencv for that. Hope that helps :)

**Comment by junyoung-jamong** [comment]
https://github.com/lzx1413/labelImgPlus
https://github.com/wkentaro/labelme

**Comment by ryouchinsa** [comment]
Using this script, you can convert the COCO segmentation format to the YOLO segmentation format.
https://github.com/ultralytics/JSON2YOLO

RectLabel is an offline image annotation tool for object detection and segmentation.
Although this is not an open source program, with RectLabel you can read/write/export the YOLO segmentation format.
https://rectlabel.com/help#xml_to_yolo
```
class_index x1 y1 x2 y2 x3 y3 ...
0 0.180027 0.287930 0.181324 0.280698 0.183726 0.270573 ...
```
![yolo_polygon](https://user-images.githubusercontent.com/1954306/214472214-a9c2775d-247a-4bd1-87f6-9d90818ef8ff.jpg)
